docs/GettingStarted.md

```diff
@@ -69,11 +69,12 @@ On the defender machine, do the following:
 9. Do `sc start EbpfCore`
 10. Do `sc create NetEbpfExt type=kernel start=boot binpath=%windir%\system32\drivers\netebpfext.sys`
 11. Do `sc start NetEbpfExt`
-12. Do `sc create ebpfsvc start= auto binpath=%windir%\system32\ebpfsvc.exe type=own`
-13. Do `sc start ebpfsvc`
-14. Do `netsh add helper %windir%\system32\ebpfnetsh.dll`
-15. Install [clang](https://github.com/llvm/llvm-project/releases/download/llvmorg-11.0.0/LLVM-11.0.0-win64.exe)
-16. Copy droppacket.c and ebpf.h to a folder (such as c:\test)
+12. Do `sc create ebpfsvc start= auto binpath=%windir%\system32\ebpfsvc.exe type=own obj= "NT AUTHORITY\LocalService" password= ""`
```

Comment:
Why not use the install option from the service binary? This would avoid typos and make this easier to setup?

Reply:
Agree. Will change the steps here.